android/app/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: "com.android.library"
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 27
+    buildToolsVersion "27.0.3"
```

Comment:
Please, see [this comment](https://github.com/Microsoft/react-native-code-push/pull/1305#issuecomment-394613176)